src/cardano/CardanoEngine.ts

```diff
@@ -342,13 +340,13 @@ export class CardanoEngine extends CurrencyEngine<
     this.addToLoop('queryTransactions', TRANSACTION_POLL_MILLISECONDS).catch(
       () => {}
     )
-    await super.startEngine()
+    await super.startEngine({ seenTxCheckpoint: this.seenTxCheckpoint })
```

Comment:
Shouldn't this be `opts`, instead of `{ seenTxCheckpoint: this.seenTxCheckpoint }`